doc/pfc_asym/PFC_Asymmetric_HLD.md

```diff
@@ -0,0 +1,391 @@
+# Asymmetric PFC Test Plan
+
+* [Overview](#Overview)
+   * [Scope](#Scope)
+   * [Testbed](#Testbed)
+* [Setup configuration](#setup-configuration)
+* [Existed modules refactoring](#existed-modules-refactoring)
+* [Python  modules to setup and run test](#python-modules-to-setup-and-run-test)
+   * [Python modules](#python-modules)
+   * [Global variables](#global-variables)
+   * [PTF test case execution](#ptf-test-case-execution)
+   * [Pytest fixtures](#pytest-fixtures)
+		* [deploy_pfc_gen](#deploy_pfc_gen)
+		* [setup](#setup)
+		* [pfc_storm_template](#pfc_storm_template)
+		* [pfc_storm_runner](#pfc_storm_runner)
+		* [enable_pfc_asym](#enable_pfc_asym)
+* [Test](#Test)
+* [Test cases](#Test-cases)
+
+## Overview
+The purpose is to test functionality of Asymmetric PFC on the SONIC based DUT, closely resembling production environment.
+
+### Scope
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is to perform functional testing of Asymmetric PFC on SONIC system. There will be reused existed PTF test suite for PFC Asymmetric which is located at ```ansible\roles\test\files\saitests\pfc_asym.py```.
+
+### Testbed
+The test will run on the following testbeds:
```

Comment:
Please also specify that RPC image is required

Reply:
Fixed.